docs/tutorials/getting-started-with-node-js.md

```diff
@@ -0,0 +1,92 @@
+---
+title: Getting Started with Node JS
+group: software-languages
+author: "@liam-b"
+---
+
+{% include /style/icon.html type="warning" %}
+Note that Node.js runs frustratingly slowly on the EV3 brick and is limited to an old version of both Node.js and npm. You may want to consider using Python or another ev3dev-supported language instead.
+{: .alert .alert-warning}
+
+<p class="lead">
+Node.js is a framework that allows you to run JavaScript as a local application. It is designed to make building servers and other asynchronous apps as easy as possible. In this tutorial, we'll show you how to get up-and-running with Node.js on ev3dev.
+</p>
```

Comment:
Just do `{: .lead}` at the end here. Kramdown automatically creates the paragraph element.